04/app.js

```diff
@@ -1 +1,11 @@
-const years = [1980, 1934, 2002, 2019];
\ No newline at end of file
+const years = [1980, 1934, 2002, 2019];
+const currentDate = new Date().getFullYear()
+
+const yearsResult = countYears(years, currentDate)
+console.log(yearsResult);
+
+function countYears(yrs, currentYr) {
```

Comment:
Jeśli ma to być `currentYr` to bym przechowywał te dane w zmiennej lokalnej (wew. funkcji). Jeśli ma to być dowolny rok to bym zmienił nazwę parametru :P